### DISCUSSION TITLE
Why are recent stable releases labeled as "Pre-release"?

### DISCUSSION THREAD

**Discussion by kkebo** [swiftwasm/swift#5415] [General]
On the SwiftWasm's [Releases page](https://github.com/swiftwasm/swift/releases), each *-RELEASE version was labeled as "Latest" until swift-wasm-5.7.1-RELEASE, but since then, it became labeled as "Pre-release". For example, [swift-wasm-5.7.1-RELEASE](https://github.com/swiftwasm/swift/releases/tag/swift-wasm-5.7.1-RELEASE) is labeled as "Latest", but [swift-wasm-5.7.3-RELEASE](https://github.com/swiftwasm/swift/releases/tag/swift-wasm-5.7.3-RELEASE) and [swift-wasm-5.8.0-RELEASE](https://github.com/swiftwasm/swift/releases/tag/swift-wasm-5.8.0-RELEASE) are labeled as "Pre-release".

It doesn't matter to me, but just out of curiosity, why it has been changed?

**Comment by kateinoigakukun** [comment]
I had to do it manually but just forgot to do it. I made 5.8.0 as the latest release now

**Reply by kkebo** [reply]
Thanks.
